ruby/inject_vs_each_with_object.md

```diff
@@ -0,0 +1,49 @@
+### Inject
```

Comment:
I think it is better to start with a h1 title like "Inject vs each_with_object" and here use the h2 title.